src/resourcepack-packager/webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const TerserPlugin = require('terser-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.ts',
+  optimization: {
+    minimize: true,
+    minimizer: [
+      new TerserPlugin({
+        terserOptions: {
+          output: {
+            preamble:
+              '/* Blockbench "resourcepack-packager" Plugin created by leopoldhub*/\n\n/** This work is licensed under the Creative Commons Attribution 4.0 International License.\nTo view a copy of this license, visit http://creativecommons.org/licenses/by/4.0/ or send a letter to Creative Commons, PO Box 1866, Mountain View, CA 94042, USA. */\n\n/* the source code is in the "src/resourcepack-packager" folder! */',
+            comments: false
+          }
+        },
+        extractComments: false
+      })
+    ]
+  },
+  module: {
+    rules: [
+      {
+        test: /\.ts$/,
+        use: 'ts-loader',
+        exclude: /node_modules/
+      }
+    ]
+  },
+  resolve: {
+    extensions: ['.ts', '.js']
+  },
+  output: {
+    filename: 'resourcepack-packager.js',
+    path: path.resolve(__dirname, '../../plugins')
+  }
```

Comment:
JSZip already ships with Blockbench. You can simply add JSZip as an external dependency, that should make the plugin a lot smaller.
https://webpack.js.org/configuration/externals/
```js
  externals: {
    jszip: 'JSZip',
  },
```